src/vivarium_gates_nutrition_optimization/components/observers.py

```diff
@@ -170,8 +170,15 @@ def setup(self, builder: Builder) -> None:
         for outcome in models.PREGNANCY_OUTCOMES:
             builder.results.register_observation(
                 name=f"pregnancy_outcome_{outcome}_count",
-                pop_filter=f'alive == "alive" and pregnancy_outcome == "{outcome}" and tracked == True',
-                requires_columns=["alive", "pregnancy_outcome"],
+                pop_filter=f'alive == "alive" and tracked == True '
+                f'and previous_pregnancy == "pregnant" and pregnancy == "parturition" '
```

Comment:
Are simulants in `parturition` for more than one time-step? If not, we don't need to look at the previous pregnancy state?

Reply:
Tagging @hussain-jafari because I am not sure and I think he made these updates

Reply:
I believe they are in the parturition state for only one time step, but I wrote the pop filter this way because this is how the birth recorder component decides when to record a birth. Otherwise, I don't have any objection to removing the previous_pregnancy check.

Reply:
Yeah, they are in parturition for only one timestep, but I am concerned about not catching the birth outcomes among mothers who died! Would removing the alive requirement do it? We determine birth outcome at initialization, I believe, so I wonder if we could just count at that point rather than trying to catch it in the sim?

Reply:
We can count at initialization. I'll go ahead and make that change.

Reply:
I think that would be great! We should probably run one final V&V check on that before production runs then though, right?